### PR TITLE
Patch release 2.1.1

### PR DESCRIPTION
**Pull request by LaurenceKuhl**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/crisprseq/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/crisprseq _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: workflows/crisprseq_screening.nf [hunk 1]
```diff
@@ -87,37 +88,59 @@ workflow CRISPRSEQ_SCREENING {
...
         // MODULE: Run FastQC
         //
         FASTQC (
             ch_input
         )
+        ch_versions = ch_versions.mix(FASTQC.out.versions.first())
 
 
-        ch_versions = ch_versions.mix(FASTQC.out.versions.first())
+        empty_channel = Channel.value([[]])
```
Comment (mirpedrol) [inline workflows/crisprseq_screening.nf:104]: I think this channel is not used
```suggestion
```

File: workflows/crisprseq_screening.nf [hunk 2]
```diff
@@ -87,37 +88,59 @@ workflow CRISPRSEQ_SCREENING {
...
             }
-            return ["${a[0]},${b[0]}", a[1] + b[1], b[2]]
+            return ["${a[0]},${b[0]}", a[1] + b[1], b[2] ,a[3] + b[3]]
         }
-        .map { condition, fastqs, single_end ->
-            [[id: condition, single_end: single_end], fastqs]
+        .map { condition, fastqs_1, single_end, fastqs_2 ->
+            [[id: condition, single_end: single_end], fastqs_1, fastqs_2]
         }
+        .last()
```
Comment (mirpedrol) [inline workflows/crisprseq_screening.nf:141]: Can be removed? If I am not wrong, this channel will always have only one element
```suggestion
```

**Review by mirpedrol — APPROVED** [review]

File: conf/test_targeted.config [hunk 3]
```diff
@@ -25,4 +25,7 @@ params {
 
     // Aligner
     aligner = 'minimap2'
+
+    // Steps
+    overrepresented = true
```
Comment (mashehu) [inline conf/test_targeted.config:30]:
```suggestion
    over_represented = true
```

File: workflows/crisprseq_screening.nf (hunk 1 above)
Comment (mashehu) [inline workflows/crisprseq_screening.nf:104]:
```suggestion
```
don't see it used anywhere

**Review by mashehu — APPROVED** [review]

File: conf/test_targeted.config (hunk 3 above)
Comment (mirpedrol) [reply conf/test_targeted.config:30]: ⚠️ the parameter is now called `overrepresented`, if we change it here we have to change it in `nextflow.config` and wherever `params.overrepresented` is used

File: workflows/crisprseq_screening.nf [hunk 4]
```diff
@@ -87,37 +88,57 @@ workflow CRISPRSEQ_SCREENING {
...
                 error "Your samplesheet contains a mix of single-end and paired-end data. This is not supported."
             }
-            return ["${a[0]},${b[0]}", a[1] + b[1], b[2]]
+            return ["${a[0]},${b[0]}", a[1] + b[1], b[2] ,a[3] + b[3]]
         }
-        .map { condition, fastqs, single_end ->
-            [[id: condition, single_end: single_end], fastqs]
+        .map { condition, fastqs_1, single_end, fastqs_2 ->
+            [[id: condition, single_end: single_end], fastqs_1, fastqs_2]
         }
```
Comment (mirpedrol) [inline workflows/crisprseq_screening.nf:139]: sorry for breaking the tests! 🙏 
```suggestion
        }
        .last()
```

**Pull request merged**